{
  try {
    Employee emp=bean.queryEmployeeName(1);
  }
 catch (  Exception e) {
    if (!(e instanceof NoResultException)) {
      Assert.fail("Expected NoResultException but got " + e);
    }
  }
}
